### ISSUE TITLE
textbox-container.floatingLabel: Wrong position if control value is set to value false

### ISSUE THREAD

**Issue by cluetjen** [telerik/kendo-angular#1496]
## I'm submitting a...
* Bug report 

## Current behavior

```
<kendo-textbox-container floatingLabel="Test">
    <kendo-dropdownlist
        [data]="[{key: false, value: 'False'}, {key: true, value: 'True'}]"
        valueField="key" textField="value" [valuePrimitive]="true"
    >
</kendo-textbox-container>
```

If "False" is selected, the label will be placed over the dropdown as if the dropdown was empty. (applies to material theme only, default theme seems to not support it)

## Expected behavior

Show floatingLabel above dropdown.

## What is the motivation or use case for changing the behavior?
Use value of type boolean with floatingLabel in material theme.

## Notes

The reason for this seems to be the following part in textbox-container.component:

```
    updateState() {
        const formControl = this.formControl;
        const empty = value => {
            // zero is not an empty value (e.g. NumericTextBox)
            if (value === 0) {
                return false;
            }
            // empty arrays are an empty value (e.g. MultiSelect)
            if (Array.isArray(value) && !value.length) {
                return true;
            }
            return !value;
        };
```

... where `return !value` seems to mark the container as "empty" if value===false

**Comment by tsvetomir** [comment]
Fixed in the latest [development version](https://www.telerik.com/kendo-angular-ui/components/installation/development-builds/).

**Comment by tsvetomir** [comment]
Shipped in `@progress/kendo-angular-inputs` v3.2.1
